bin/lsst-build

```diff
@@ -57,5 +57,6 @@ build.add_argument(
 )
 
 args = parser.parse_args()
+args.products = args.products[0].split()
```

Comment:
This looks wrong to me. What about the other contents of `args.products` ? Does `rebuild lsst_distrib lsst_ci` work with this change? That worked before and `products` is setup with `nargs+`. Do we know what lsstsw does with the PRODUCTS configuration option that makes it fail? Does it quote it or something?

Reply:
@timj 

I think you're right that changing the Python CLI is probably not the right place to fix this...

If I instead change the following in the rebuild script itself within lsstsw, then multiple products are handled properly by Python with needing to change the CLI.

```
-  ARGS+=("${PRODUCTS[@]}")
+  ARGS+=(${PRODUCTS[@]})
```

```
-  PRODUCTS=("$@")
+  PRODUCTS=($@)
```

I checked that `rebuild lsst_distrib lsst_ci` works when quotes are removed, as does `rebuild` by itself (which is now building just the single `lsst_distrib` product).


Does removing the quotes have any other effects that would be undesirable?